modules/postgresql/outputs.tf

```diff
@@ -152,3 +152,11 @@ output "env_vars" {
     "CLOUD_SQL_DATABASE_NAME" : local.database_name
   }
 }
+
+output "apphub_service_uri" {
+  value = {
+    service_uri = "//cloudsql.googleapis.com/projects${element(split("/projects", google_sql_database_instance.default.self_link), 1)}"
+    service_id  = substr(format("%s-%s", var.name, md5(var.project_id)), 0, 63)
```

Comment:
Let's also include var.region while creating the service_id as is done elsewhere.